solr/core/src/java/org/apache/solr/handler/admin/LukeRequestHandler.java

```diff
@@ -298,7 +300,7 @@ private static String getFieldFlags( SchemaField f )
 
       BytesRef bytes = field.binaryValue();
       if (bytes != null) {
-        f.add( "binary", Base64.byteArrayToBase64(bytes.bytes, bytes.offset, bytes.length));
+        f.add( "binary", new String(Base64.getEncoder().encode(ByteBuffer.wrap(bytes.bytes, bytes.offset, bytes.length)).array(), StandardCharsets.ISO_8859_1));
```

Comment:
Can be simplified to `Base64.getEncoder().encodeToString(ByteBuffer.wrap(bytes.bytes, bytes.offset, bytes.length).array())`

Reply:
I'm afraid that's not exactly the same. ByteBuffer's `array()` method returns the full array and ignores the offset and the length.